linux/opt/sage/bin/make_env_vars_file.sh

```diff
@@ -32,6 +32,7 @@ extract_tag_value() {
 DEPARTMENT=$(extract_tag_value Department)
 PROJECT=$(extract_tag_value Project)
 PROVISIONING_PRINCIPAL_ARN=$(extract_tag_value 'aws:servicecatalog:provisioningPrincipalArn')
+ACCESS_APPROVED_ROLEID=$(extract_tag_value AccessApprovedRoleId)
```

Comment:
any reason why you changed name from PRODUCT_ACCESS_APPROVED_ROLEID to ACCESS_APPROVED_ROLEID?

Reply:
I changed it because it uses the ec2 describe-tags query rather than the product query.